### DISCUSSION TITLE
Cómo obtener el developer para comandos de Discord

### DISCUSSION THREAD

**Discussion by ITXJ0n** [is-a-dev/register#15003] [Support] [answered]
Tengo un bot y me gustaría poder saber si podrían aceptarme ya que es para un servidor de roleplay OCZ PS4/PS4 Es una comunidad con casi 10000 personas alrededor de 9700 tenemos ahora

**Comment by wdhdev** [comment, marked as answer]
I'm not really sure what you're asking here. Are you asking if you can have a subdomain for your Discord bot?
